src/components/WwwFrame/TheHeader.vue

```diff
@@ -762,29 +762,29 @@ export default {
 		},
 		{
 			query: experimentAssignmentQuery,
-			preFetch: true,
-			variables() {
-				return {
-					id: NAV_UPDATE_EXP_KEY
-				};
+			preFetch(_, client) {
+				return client.query({
```

Comment:
This isn't going to work for pages that use cacheable SSR (currently only UI-Site-Map). @christian14b @dyersituations How long do we expect to run this experiment? if it's going to be for a while, this experiment assignment should be handled with Fastly instead and the tracking should only happen in the `mounted` callback.